bust-a-move-framework/src/test/java/bustamove/AppTest.java

```diff
@@ -21,8 +21,6 @@ public void setUp() {
     @Test
     public void setgetGame() {
         Game game = new Game();
-        assertNull(app.getGame());
-        app.setGame(game);
         assertEquals(game, app.getGame());
```

Comment:
Why should game and the game in app be the same? what is the purpose of this test now?